src/components/Clock.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  today: Date;
+};
+
+function getTime(date: Date): string {
+  return date.toUTCString().slice(-12, -4);
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  clockId = 0;
+
+  componentDidMount(): void {
+    this.clockId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(
+        getTime(this.state.today),
+      );
```

Comment:
Don't use state here, it can be outdated
Save `new Date()` result to constant and use if for both `setState` and `console.info`